Source/SwiftLintBuiltInRules/Rules/Lint/NonOptionalStringDataConversionRule.swift

```diff
@@ -5,14 +5,16 @@ struct NonOptionalStringDataConversionRule: Rule {
     var configuration = SeverityConfiguration<Self>(.warning)
     static let description = RuleDescription(
         identifier: "non_optional_string_data_conversion",
-        name: "Non-optional String -> Data Conversion",
-        description: "Prefer non-optional `Data(_:)` initializer when converting `String` to `Data`",
+        name: "Non-optional String <-> Data Conversion",
+        description: "Prefer the non-optional initializers when converting between `String` and `Data` (e.g. `Data(_:)` and `String(decoding:as:)`)",
```

Comment:
If you are still interested in providing extended documentation for this rule, please let the `description` property as it is and prefer the optional `rationale` property. It serves exactly this purpose.